cmd/lotus-chainwatch/storage.go

```diff
@@ -252,6 +254,24 @@ create table if not exists receipts
 
 create index if not exists receipts_msg_state_index
 	on receipts (msg, state);
+	
+create table if not exists miner_sectors
+(
+	stateroot text not null,
+	miner text not null,
+	sectorid bigint not null,
+	activation bigint not null,
+	dealweight bigint not null,
+	verifieddealweight bigint not null,
+	expiration bigint not null,
+	sealcid text not null,
+	sealrandepoch bigint not null,
+	constraint miner_sectors_pk
+		primary key (stateroot, miner, sectorid)
+);
+
+create index if not exists miner_sectors_state_index
+	on miner_sectors (stateroot, miner, sectorid);
```

Comment:
This index name should be something like `miner_sectors_stateroot_miner_sectorid_index`.